.github/workflows/CI.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           - x64
     steps:
       - uses: actions/checkout@v2
+      - run:  perl -ni -e 'print unless /GLMakie/' ext/Project.toml
```

Comment:
Why do we need this?

Reply:
This removes `GLMakie` from the weak dependencies environment. It's just a sort of optimization: since it is the heaviest package we have and we don't need it in CI, this saves some time while instantiating the environment.